hardhat.config.ts

```diff
@@ -117,67 +107,60 @@ task("deploy:contracts-only", "Deploy the set of bare contracts").setAction(
 );
 
 task("roles:set-default", "Set default roles on all deployed contracts")
-  .addOptionalParam("license", "ERC721 License used to find owners", undefined, types.json)
```

Comment:
These parameters are still needed for the zkSync deployment. They just don't need to be used for the normal deployment.

Reply:
ahh i see - ok i added those back

Reply:
ah ok i see this now -  ive removed the other dependencies on the zk deploy for contracts (passing address) and I also fixed some the argument passing -- this is what my deploy script looks like now outside of zk:

```
> npx hardhat deploy --network local --show-stack-traces

Deploying all contracts...
GeoWebCoordinate deployed to: 0x51C65cd0Cdb1A8A8b79dfc2eE965B1bA0bb8fc89
GeoWebCoordinatePath deployed to: 0x8fC8CFB7f7362E44E472c690A6e025B80E406458
GeoWebParcel deployed to: 0xC7143d5bA86553C06f5730c8dC9f8187a621A8D4
Accountant deployed to: 0x359570B3a0437805D0a71457D61AD26a28cAC9A2
ERC721License deployed to: 0xc9952Fc93Fa9bE383ccB39008c786b9f94eAc95d
ETHPurchaser deployed to: 0xDde063eBe8E85D666AD99f731B4Dbf8C98F29708
ETHExpirationCollector deployed to: 0xD5724171C2b7f0AA717a324626050BD05767e2C6
SimpleETHClaimer deployed to: 0x70eE76691Bdd9696552AF8d4fd634b3cF79DD529
Contracts deployed.

Setting default configuration...
Successfully set Accountant fee.
Successfully set Accountant validator.
Successfully set ETHExpirationCollector minContributionRate.
Successfully set ETHExpirationCollector minExpiration.
Successfully set ETHExpirationCollector maxExpiration.
Successfully set ETHExpirationCollector license.
Successfully set ETHExpirationCollector receiver.
Successfully set ETHExpirationCollector accountant.
Successfully set ETHPurchaser dutchAuctionLengthInSeconds.
Successfully set ETHPurchaser license.
Successfully set ETHPurchaser accountant.
Successfully set ETHPurchaser collector.
Successfully set SimpleETHClaimer claimer.
Successfully set SimpleETHClaimer license.
Successfully set SimpleETHClaimer parcel.
Successfully set SimpleETHClaimer collector.
Default configuration set.

Setting roles...
Successfully granted Accountant.MODIFY_CONTRIBUTION_ROLE to ETHExpirationCollector
Default roles set.
```

that actually seems like it covers the full deploy script vs whats in the description.

so sorry for spending so much time on this pr and correct me if im wrong here - it sort of feels like I should be doing the opposite now that I look at this - returning contracts from the standard deploy and allowing the zk deploy to remain the same (basing the scripts on returning contracts instead of addresses)

want me to reverse the logic? or are we good with just the address being passed?

Reply:
I think this is good for now. But later let's work together on the zkSync deployment and we can revisit what is best here.